features/support/examples/rspec/spec_helper_only_rack_test.rb.example

```diff
@@ -3,7 +3,6 @@ require "rspec/core"
 require "rspec/matchers"
 require "rspec/expectations"
 require "raml-rb"
-require "rack/test"
 
 path = File.expand_path('../../lib', __FILE__)
 
```

Comment:
I already created the rambo_helper idea. From that we can add any requires we need and it will be more flexible going forward.